### PR TITLE
updated architecture

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- second attempt to provide an updated architecture for oVirt

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @oVirt/developers

### CODE REVIEW

File: source/images/diagrams/ovirt-architecture.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+with Diagram(
+    "oVirt Architecture",
+    direction="TB",
+    graph_attr={"nodesep": "1", "ranksep": "1", "splines": "compound"}
+):
+
+    with Cluster("Storage Domains"):
+        gluster = Glusterfs("Gluster storage")
+        iscsi = Rack("ISCS Storage")
```
Comment (michalskrivanek) [inline source/images/diagrams/ovirt-architecture.py:32]: iSCSI

also, would it be possible to add ceph? maybe one more box with cinderlib on host going to a ceph "storage box"

Comment (sandrobonazzola) [reply source/images/diagrams/ovirt-architecture.py]: Done both, let me know if seems good enough.

**Review by avlitman — COMMENTED** [review]
Suggestion: on source/images/diagrams/ovirt_architecture.png I might connect the postgres engine db directly to postgres dwh db and write over the arrow between them: aggregations.

image

Other than that LGTM.

**Review by ljelinkova — COMMENTED** [review]
The section _Web-based User Interface_ (not updated in this PR) is relevant only for the Admin Portal, the Web UI Portal is based on React, not GWT. @rszwajko @sgratch and @sjd78

File: source/develop/architecture/architecture.md [hunk 2]
```diff
@@ -9,54 +9,53 @@ authors:
...
 
-![](/images/wiki/Ovirt-1024x698.png "Ovirt-1024x698.png")
 
 # oVirt Architecture
 
+![](/images/diagrams/ovirt_architecture.png "oVirt architecture")
+
 A standard oVirt deployment consists of three things, primarily:
 
 *   ovirt-engine which is used use to deploy, monitor, move, stop and create VM images, configure storage, network , etc.
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/develop/architecture/architecture.md:24]: Thanks!

File: source/develop/architecture/architecture.md [hunk 3]
```diff
@@ -9,54 +9,53 @@ authors:
...
 *   One or more hosts (nodes), on which we run virtual machines (VMs)
 *   One or more storage nodes, which hold the images and ISOs corresponding to those VMs
 
-Also, usually an identity service is deployed aside the engine, to authenticate users and administrators for ovirt-engine.
-
-The nodes are Linux distributions with VDSM and libvirt installed, along with some extra packages to easily enable virtualization of networking and other system services. The supported Linux distributions to date are Fedora 17 or oVirt-node, which is basically a stripped-down distribution containing just enough components to allow virtualization.
+The nodes are Linux distributions with VDSM and libvirt installed, along with some extra packages to easily enable virtualization of networking and other system services.
+The supported Linux distributions to date are CentOS Stream 8 and derivatives. oVirt project also providesoVirt Node, which is basically a stripped-down CentOS Stream 8 distribution
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/develop/architecture/architecture.md:29]: Thanks!

**Comment by sandrobonazzola** [comment]
In reply to [@avlitman comment](https://github.com/oVirt/ovirt-site/pull/3019#pullrequestreview-1035781688)

> connect the postgres engine db directly to postgres dwh db and write over the arrow between them: aggregations

I tried that and it disrupted the whole diagram. I would rather leave this detail level to DWH/Grafana specific architecture diagram.

**Review by avlitman — APPROVED** [review]

**Review by ljelinkova — APPROVED** [review]

**Review by michalskrivanek — APPROVED** [review]

**Pull request merged**